pkgs/development/libraries/libkeyfinder/default.nix

```diff
@@ -1,37 +1,32 @@
-{ lib, stdenv, fetchFromGitHub, fftw, qtbase, qmake }:
+{ lib, stdenv, fetchFromGitHub, cmake, fftw, catch2 }:
 
 stdenv.mkDerivation rec {
   pname = "libkeyfinder";
-  version = "2.1";
+  version = "2.2.4";
 
   src = fetchFromGitHub {
-    sha256 = "07kc0cl6kirgmpdgkgmp6r3yvyf7b1w569z01g8rfl1cig80qdc7";
+    owner = "mixxxdj";
+    repo = "libkeyfinder";
     rev = "v${version}";
-    repo = "libKeyFinder";
-    owner = "ibsh";
+    sha256 = "005qq81xfzi1iifvpgkqpizxcrfisafq2r0cjp4fxqh1ih7bfimv";
   };
 
-  nativeBuildInputs = [ qmake ];
-  buildInputs = [ fftw qtbase ];
-
-  postPatch = ''
-    substituteInPlace LibKeyFinder.pro \
-      --replace "/usr/local" "$out" \
-      --replace "-stdlib=libc++" ""
+  # needed for linking libkeyfinder.so into keyfinder-tests executable
```

Comment:
This can be usually resolved using `cmakeFlags = [ "-DCMAKE_SKIP_BUILD_RPATH=OFF" ];`.

Reply:
Thanks! Fixed in 85e8220aa70ba637853cb31232a4c12c96760839.